### DISCUSSION TITLE
Is 4.x considered to be in a code freeze?

### DISCUSSION THREAD

**Discussion by kara-todd** [lodash/lodash#5725] [General]
I read the about the `issue bankruptcy` label and completely understand.

However, there are likely still some key issues around 4.x version of the library. Previously, I opened an issue about a high security warning from Synk ( #5499 ) that prevented me from being able to use lodash.set on another project. I don't want to introduce clutter to the issues column again, but what is the best way to deal with these issues? Should we wait for the 5.x branch to address these issues?

Thanks for the great library!

**Comment by Shinigami92** [comment]
- https://github.com/lodash/lodash/issues/5719#issuecomment-1722536384

> Lodash 4 will get surgical, handcrafted updates that will aim to preserve existing compat.

**Comment by jdalton** [comment]
Hi @kara-todd! 

Lodash v4 will get updates but just for bug/security fixes. 
Feature work will be in v5.

To better represent the state of the world I may create a new branch that has snapshots of packages (npm tars extracted). So they can be manually worked on.
